diff --git a/runtime/Dockerfile b/runtime/Dockerfile
index 6a08cec92367e6a6f8e51d480801a9f18f9f3ec2..2d00ca5f47c8a8aeca113183947d6a71d4d851c4 100644
--- a/runtime/Dockerfile
+++ b/runtime/Dockerfile
@@ -2,8 +2,12 @@ FROM node:22-bookworm-slim AS dependencies
 WORKDIR /app
 RUN corepack enable
 COPY package.json pnpm-lock.yaml pnpm-workspace.yaml ./
+COPY patches ./patches
 RUN pnpm install --prod --frozen-lockfile
 FROM node:22-bookworm-slim
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends curl \
+    && rm -rf /var/lib/apt/lists/*
 WORKDIR /app
 ARG PORTAL_CONFIG=portal.config.json
 ENV CSP_CONFIG_PATH=$PORTAL_CONFIG
